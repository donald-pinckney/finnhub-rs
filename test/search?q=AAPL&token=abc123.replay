{"request":{"url":"https://finnhub.io/api/v1/search?q=AAPL&token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/search?q=AAPL&token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"644c3cf77f913fc7-ORD","cf-request-id":"09a3846eb000003fc75297e000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Sat, 24 Apr 2021 03:27:51 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"max_age\":604800,\"report_to\":\"cf-nel\"}","report-to":"{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report?s=yDWm0zQJ%2FoRoHQA79Q5B3wNklCtL60D7%2F%2BfhSjowl6rqG1LGHFRDchibgroFBmsjEnkBUKo1GHRwgQZBXedt%2FsaTiVy8vaYxOqlBTworMpeaeWix9zqo\"}],\"max_age\":604800,\"group\":\"cf-nel\"}","server":"cloudflare","set-cookie":"__cfduid=d4fa34043f81174a7c6d5eec996cb7d811619234870; expires=Mon, 24-May-21 03:27:50 GMT; path=/; domain=.finnhub.io; HttpOnly; SameSite=Lax; Secure","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"47","x-ratelimit-reset":"1619234930"},"body":"eyJjb3VudCI6MTgsInJlc3VsdCI6W3siZGVzY3JpcHRpb24iOiJBUFBMRSBJTkMiLCJkaXNwbGF5U3ltYm9sIjoiQUFQTCIsInN5bWJvbCI6IkFBUEwiLCJ0eXBlIjoiQ29tbW9uIFN0b2NrIn0seyJkZXNjcmlwdGlvbiI6IkFQUExFIElOQyIsImRpc3BsYXlTeW1ib2wiOiJBQVBMLlNXIiwic3ltYm9sIjoiQUFQTC5TVyIsInR5cGUiOiJDb21tb24gU3RvY2sifSx7ImRlc2NyaXB0aW9uIjoiQVBQTEUgSU5DIiwiZGlzcGxheVN5bWJvbCI6IkFBUEwuU04iLCJzeW1ib2wiOiJBQVBMLlNOIiwidHlwZSI6IkNvbW1vbiBTdG9jayJ9LHsiZGVzY3JpcHRpb24iOiJBUFBMRSBJTkMiLCJkaXNwbGF5U3ltYm9sIjoiQUFQTC5WSSIsInN5bWJvbCI6IkFBUEwuVkkiLCJ0eXBlIjoiQ29tbW9uIFN0b2NrIn0seyJkZXNjcmlwdGlvbiI6IkFQUExFIElOQyIsImRpc3BsYXlTeW1ib2wiOiJBQVBMLk1JIiwic3ltYm9sIjoiQUFQTC5NSSIsInR5cGUiOiJDb21tb24gU3RvY2sifSx7ImRlc2NyaXB0aW9uIjoiQVBQTEUgSU5DIiwiZGlzcGxheVN5bWJvbCI6IkFBUEwuQkMiLCJzeW1ib2wiOiJBQVBMLkJDIiwidHlwZSI6IkNvbW1vbiBTdG9jayJ9LHsiZGVzY3JpcHRpb24iOiJBQSBQTEMiLCJkaXNwbGF5U3ltYm9sIjoiQUFUREYiLCJzeW1ib2wiOiJBQVRERiIsInR5cGUiOiJDb21tb24gU3RvY2sifSx7ImRlc2NyaXB0aW9uIjoiUGFtYXBvbCIsImRpc3BsYXlTeW1ib2wiOiIwTFhHLkwiLCJzeW1ib2wiOiIwTFhHLkwiLCJ0eXBlIjoiIn0seyJkZXNjcmlwdGlvbiI6IkF2aXZhIFBsYyIsImRpc3BsYXlTeW1ib2wiOiIzNFhYLkwiLCJzeW1ib2wiOiIzNFhYLkwiLCJ0eXBlIjoiIn0seyJkZXNjcmlwdGlvbiI6IkF2aXZhIFBsYyIsImRpc3BsYXlTeW1ib2wiOiI3N0VCLkwiLCJzeW1ib2wiOiI3N0VCLkwiLCJ0eXBlIjoiIn0seyJkZXNjcmlwdGlvbiI6IkF2aXZhIFBsYyIsImRpc3BsYXlTeW1ib2wiOiI1OUtGLkwiLCJzeW1ib2wiOiI1OUtGLkwiLCJ0eXBlIjoiIn0seyJkZXNjcmlwdGlvbiI6IkF2aXZhIFBsYyIsImRpc3BsYXlTeW1ib2wiOiIxMUdWLkwiLCJzeW1ib2wiOiIxMUdWLkwiLCJ0eXBlIjoiIn0seyJkZXNjcmlwdGlvbiI6Ik1ha2Fyb24gUGwiLCJkaXNwbGF5U3ltYm9sIjoiMExWUS5MIiwic3ltYm9sIjoiMExWUS5MIiwidHlwZSI6IiJ9LHsiZGVzY3JpcHRpb24iOiJBdml2YSBQbGMiLCJkaXNwbGF5U3ltYm9sIjoiNTFJRS5MIiwic3ltYm9sIjoiNTFJRS5MIiwidHlwZSI6IiJ9LHsiZGVzY3JpcHRpb24iOiJBTExBIFBDTCIsImRpc3BsYXlTeW1ib2wiOiJBTExBLkJLIiwic3ltYm9sIjoiQUxMQS5CSyIsInR5cGUiOiJDb21tb24gU3RvY2sifSx7ImRlc2NyaXB0aW9uIjoiU0FHQSBQTEMiLCJkaXNwbGF5U3ltYm9sIjoiNjVKLk1VIiwic3ltYm9sIjoiNjVKLk1VIiwidHlwZSI6IkNvbW1vbiBTdG9jayJ9LHsiZGVzY3JpcHRpb24iOiJBdml2YSBQTEMiLCJkaXNwbGF5U3ltYm9sIjoiQVYuQS5MIiwic3ltYm9sIjoiQVYuQS5MIiwidHlwZSI6IiJ9LHsiZGVzY3JpcHRpb24iOiJCYXJjbGF5cyBwbGMiLCJkaXNwbGF5U3ltYm9sIjoiMzFQRC5MIiwic3ltYm9sIjoiMzFQRC5MIiwidHlwZSI6IiJ9XX0="},"format_version":3}