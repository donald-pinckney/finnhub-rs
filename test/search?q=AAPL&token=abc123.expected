SymbolLookup {
    count: 18,
    result: [
        Results {
            description: "APPLE INC",
            displaySymbol: "AAPL",
            symbol: "AAPL",
            _type: "Common Stock",
        },
        Results {
            description: "APPLE INC",
            displaySymbol: "AAPL.SW",
            symbol: "AAPL.SW",
            _type: "Common Stock",
        },
        Results {
            description: "APPLE INC",
            displaySymbol: "AAPL.SN",
            symbol: "AAPL.SN",
            _type: "Common Stock",
        },
        Results {
            description: "APPLE INC",
            displaySymbol: "AAPL.VI",
            symbol: "AAPL.VI",
            _type: "Common Stock",
        },
        Results {
            description: "APPLE INC",
            displaySymbol: "AAPL.MI",
            symbol: "AAPL.MI",
            _type: "Common Stock",
        },
        Results {
            description: "APPLE INC",
            displaySymbol: "AAPL.BC",
            symbol: "AAPL.BC",
            _type: "Common Stock",
        },
        Results {
            description: "AA PLC",
            displaySymbol: "AATDF",
            symbol: "AATDF",
            _type: "Common Stock",
        },
        Results {
            description: "Pamapol",
            displaySymbol: "0LXG.L",
            symbol: "0LXG.L",
            _type: "",
        },
        Results {
            description: "Aviva Plc",
            displaySymbol: "34XX.L",
            symbol: "34XX.L",
            _type: "",
        },
        Results {
            description: "Aviva Plc",
            displaySymbol: "77EB.L",
            symbol: "77EB.L",
            _type: "",
        },
        Results {
            description: "Aviva Plc",
            displaySymbol: "59KF.L",
            symbol: "59KF.L",
            _type: "",
        },
        Results {
            description: "Aviva Plc",
            displaySymbol: "11GV.L",
            symbol: "11GV.L",
            _type: "",
        },
        Results {
            description: "Makaron Pl",
            displaySymbol: "0LVQ.L",
            symbol: "0LVQ.L",
            _type: "",
        },
        Results {
            description: "Aviva Plc",
            displaySymbol: "51IE.L",
            symbol: "51IE.L",
            _type: "",
        },
        Results {
            description: "ALLA PCL",
            displaySymbol: "ALLA.BK",
            symbol: "ALLA.BK",
            _type: "Common Stock",
        },
        Results {
            description: "SAGA PLC",
            displaySymbol: "65J.MU",
            symbol: "65J.MU",
            _type: "Common Stock",
        },
        Results {
            description: "Aviva PLC",
            displaySymbol: "AV.A.L",
            symbol: "AV.A.L",
            _type: "",
        },
        Results {
            description: "Barclays plc",
            displaySymbol: "31PD.L",
            symbol: "31PD.L",
            _type: "",
        },
    ],
}
